node/tests/utils.rs

```diff
@@ -359,10 +361,15 @@ impl MASQNode {
         ensure_start: bool,
         command_getter: F,
     ) -> MASQNode {
-        let data_dir = if sterile_database {
-            ensure_node_home_directory_exists("integration", test_name)
-        } else {
-            node_home_directory("integration", test_name)
+        running_test();
```

Comment:
Do you know why you put this line in here? I can't come up with a good argument for keeping it. Please take it seriously and think it out thoroughly. If you have troubles to believe it is needed you probably should remove it.